bitpay/wallet
# Copay Native Binaries Build

Scripts to generate native binary installers for Copay. (5f2262e and be16df5)

It is very OSX centric, but since it is scripted in node the bulk of it should be re-usable if anybody wants to tweak it to make it run on other platforms.

There's also a little change to the way closing windows works in OSX just to make it a little more coherent in preparation for rolling it out to users. (ebfaa24)

To test it after merge:
npm install
brew install makensis
npm run dist

Unless the binary atom-shell downloads fail (which has been known to happen occasionally) you should end up with binaries in dist after running the above.
